src/lib/coda_base/account.ml

```diff
@@ -163,58 +168,56 @@ let typ : (var, value) Typ.t =
     ; Nonce.Unpacked.typ
     ; Receipt.Chain_hash.typ
     ; Public_key.Compressed.typ
-    ; Boolean.typ ]
+    ; State_hash.typ ]
   in
   let of_hlist
         : 'a 'b 'c 'd 'e.    ( unit
                              , 'a -> 'b -> 'c -> 'd -> 'a -> 'e -> unit )
                              H_list.t -> ('a, 'b, 'c, 'd, 'e) Poly.t =
     let open H_list in
-    fun [public_key; balance; nonce; receipt_chain_hash; delegate; participated]
-        ->
-      {public_key; balance; nonce; receipt_chain_hash; delegate; participated}
+    fun [public_key; balance; nonce; receipt_chain_hash; delegate; voting_for] ->
+      {public_key; balance; nonce; receipt_chain_hash; delegate; voting_for}
   in
   let to_hlist
       Poly.
-        {public_key; balance; nonce; receipt_chain_hash; delegate; participated}
+        {public_key; balance; nonce; receipt_chain_hash; delegate; voting_for}
       =
     H_list.
-      [public_key; balance; nonce; receipt_chain_hash; delegate; participated]
+      [public_key; balance; nonce; receipt_chain_hash; delegate; voting_for]
   in
   Typ.of_hlistable spec ~var_to_hlist:to_hlist ~var_of_hlist:of_hlist
     ~value_to_hlist:to_hlist ~value_of_hlist:of_hlist
 
 let var_of_t
-    ({public_key; balance; nonce; receipt_chain_hash; delegate; participated} :
+    ({public_key; balance; nonce; receipt_chain_hash; delegate; voting_for} :
       value) =
   { Poly.public_key= Public_key.Compressed.var_of_t public_key
   ; balance= Balance.var_of_t balance
   ; nonce= Nonce.Unpacked.var_of_value nonce
   ; receipt_chain_hash= Receipt.Chain_hash.var_of_t receipt_chain_hash
   ; delegate= Public_key.Compressed.var_of_t delegate
-  ; participated= Boolean.var_of_value participated }
+  ; voting_for= State_hash.var_of_t voting_for }
 
 let var_to_triples
-    Poly.
-      {public_key; balance; nonce; receipt_chain_hash; delegate; participated}
+    Poly.{public_key; balance; nonce; receipt_chain_hash; delegate; voting_for}
     =
-  let%map public_key = Public_key.Compressed.var_to_triples public_key
+  let%bind public_key = Public_key.Compressed.var_to_triples public_key
```

Comment:
It doesn't matter but just fyi you can include the `voting_for` in the `%map` using `and`.